package-lock.json

```diff
@@ -4061,8 +4061,7 @@
     "graphql": {
       "version": "15.3.0",
       "resolved": "https://registry.npmjs.org/graphql/-/graphql-15.3.0.tgz",
-      "integrity": "sha512-GTCJtzJmkFLWRfFJuoo9RWWa/FfamUHgiFosxi/X1Ani4AVWbeyBenZTNX6dM+7WSbbFfTo/25eh0LLkwHMw2w==",
-      "dev": true
```

Comment:
This was changed in a previous PR (devDependencies -> dependencies) but we didn't update package-lock for it